LightBlue/Setup/LightBlueConfiguration.cs

```diff
@@ -51,9 +51,25 @@ public static void SetAsMultiHost()
             {
                 throw new InvalidOperationException("LightBlue has already been initialised and cannot be reconfigured");
             }
+
             _context = new LightBlueLogicalCallContext();
         }
 
+        public static string SetAsWindowsHost(string service, string cscfg, string csdef, string roleName)
+        {
+            StandaloneEnvironment.LightBlueDataDirectory = @"c:\ProgramData\LightBlue";
```

Comment:
Changes the the data directory to a location that doesn't change depending on what user the service is running under